usr/share/inputplumber/devices/25-playtron-suiplay_0x1.yaml

```diff
@@ -0,0 +1,104 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/ShadowBlip/InputPlumber/main/rootfs/usr/share/inputplumber/schema/composite_device_v1.json
+# Schema version number
+version: 1
+
+# The type of configuration schema
+kind: CompositeDevice
+
+# Name of the composite device mapping
+name: SuiPlay 0x1
+
+# Only allow a single source device per composite device of this type.
+single_source: false
+
+# Only use this profile if *any* of the given matches matches. If this list is
+# empty, then the source devices will *always* be checked.
+# /sys/class/dmi/id/product_name
+matches:
+  - dmi_data:
+      product_name: SuiPlay0X1
+
+# One or more source devices to combine into a single virtual device. The events
+# from these devices will be watched and translated according to the key map.
+source_devices:
+  - group: gamepad
+    evdev:
+      name: Nintendo Co., Ltd. Pro Controller
+      phys_path: usb-0000:c4:00.3-4/input0
+      handler: event*
+  - group: gamepad
+    evdev:
+      name: Microsoft X-Box 360 pad
+      phys_path: usb-0000:c4:00.3-4/input0
+      handler: event*
+  - group: keyboard
+    evdev:
+      name: AT Translated Set 2 keyboard
+      phys_path: isa0060/serio0/input0
+      handler: event*
+    events:
+      exclude:
+        - "*"
+      include:
+        - Keyboard:KeyVolumeUp
+        - Keyboard:KeyVolumeDown
+        - Keyboard:KeyLeftMeta
+        - Keyboard:KeyRightCtrl
+        - Keyboard:KeyD
+        - Keyboard:KeyF15
+        - Keyboard:KeyF16
+        - Keyboard:KeyF17
+        - Keyboard:KeyF18
+  - group: imu
+    iio:
+      name: i2c-BMI0160:00
```

Comment:
Does this need to be BMI260 (or BMI270)?

Reply:
On the ayaneo 2s, it shows up with the string BMI0160, when it is actually a BMI270. It's an issue with the hardware advertising itself as the wrong model.

We will probably need to change this in the future anyway once we get the name string from the BMI323.